samples/Nancy.Demo.Async/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "version": "2.0.0-*",
+
+  "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",

```

Comment:
version probably doesn't matter. Just for OCD might be best to keep authors, description etc